examples/httpsServer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Legionth\React\Http\HttpServer;
+use React\Socket\Server as Socket;
+use RingCentral\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+use React\Socket\SecureServer;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$loop = React\EventLoop\Factory::create();
+
+$callback = function (RequestInterface $request) {
+    $content = '<html>
+<body>
+    <h1> Hello World! </h1>
+    <p> This is your own little HTTPS server. Written in PHP :-) </p>
+</body>
+</html>';
+
+    return new Response(
+        200,
+        array(
+            'Content-Length' => strlen($content),
+            'Content-Type' => 'text/html'
+        ),
+        $content
+        );
+};
+
+$socket = new Socket($loop);
+$socket = new SecureServer($socket, $loop,
+    array(
+        'local_cert' => 'server.pem',
+        'local_pk' => 'server.key'
```

Comment:
How can I run this example? Where do I get these files from?

May need (just?) some documentation, perhaps a link to the Socket component example that generates the necessary certs?